src/extension/features/accounts/reconcile-balance/index.ts

```diff
@@ -49,37 +50,39 @@ export class ReconcileBalance extends Feature {
     }
   }
 
-  observe(changedNodes) {
+  observe(changedNodes: Set<string>) {
     if (!this.shouldInvoke()) return;
 
+    console.log(changedNodes);
+
     // When the reconciled balance icon changes, reevaluate our balance
-    if (changedNodes.has('is-reconciled-icon svg-icon lock')) {
+    if (
+      changedNodes.has('is-reconciled-icon svg-icon lock') ||
+      changedNodes.has('accounts-header-reconcile button')
```

Comment:
we use `modal-account-reconcile-reconciled` in reconcile-confetti, wonder if that's a better analogue for "reconcile finished" than just observing a button?

Reply:
Fixed on https://github.com/toolkit-for-ynab/toolkit-for-ynab/commit/9ec32cb0c39de07945e1c4cf5b2246b250c6823d.